<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductListWidget</class>
 <widget class="QWidget" name="ProductListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>25</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(52, 120, 221), stop:1 rgb(67,129,162));
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1,2">
   <item>
    <widget class="QLabel" name="smallPic">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>45</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background:transparent;
border:0px;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
     <item>
      <widget class="QLabel" name="currentPriceLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
color:rgb(255, 44, 25)
}
QLabel{
background:transparent;
}</string>
       </property>
       <property name="text">
        <string>￥99999</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="priceLabel">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <strikeout>true</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
color:rgb(172, 219, 237)
}
QLabel{
background:transparent;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
